x/programs/rust/wasmlanche-sdk/src/program.rs

```diff
@@ -24,22 +25,14 @@ pub enum ExternalCallError {
 /// Represents the current Program in the context of the caller. Or an external
 /// program that is being invoked.
 #[cfg_attr(feature = "debug", derive(Debug))]
+#[derive(Clone, BorshSerialize, Serialize)]
```

Comment:
I will back off this `Serialize` add. I wanted to be able to pass the `Context` but it was not really easy to construct it in a nice way because of traits such as `Default` for `Program<K>`. Now that we have the `TestContext` we have a clean builder API.